README.md

```diff
@@ -181,29 +181,52 @@ it into the "Index pattern" field, then click the "Next step" button.
 Discover section.
 
 
-#### Using Docker
+#### Using Docker Locally
 
-1. Download, install, and launch [Docker]
+1. Download, install, and launch [Docker](https://www.docker.com/products/docker-desktop)
 
-1. Set up the Docker image
+1. Build and launch the Docker containers: `docker-compose up`
 
-  ```
-  $ bin/setup --docker
-  ```
+1. Copy necessary config files locally. This dir is mounted as a volume, so new files also appear in the container.
```

Comment:
There are still some manual steps here that can be automated, just getting things to work for now.
For these copied config files, if we do it during container build time, then once `docker-compose` comes up and mounts the entire project as a `volume`, it all gets blown away.  That made me crazy for a minute.

Reply:
For this PR, are you planning to leave in the manual steps? Or are those up for comment/review too?

Reply:
I'd like to automate as much as possible, this is just a first cut.  If you have ideas, feedback, or anything else I'd love to hear!

Reply:
I'd like to automate as much as possible. This is just a first cut. If you have any ideas/feedback/whatever I'd love to hear it!